app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -437,7 +437,7 @@ class AdministratorControlsActivityTest {
   @Test
   // TODO(#973): Fix AdministratorControlsActivityTest
   @Ignore
-  fun testAdministratorControlsFragment_loadFragment_clickEditProfile_checkOpensProfileListActivity() { // ktlint-disable max-line-length
+  fun testAdminCtrls_clickEditProfile_checkOpensProfileListActivity() {
```

Comment:
Just: ``testActivity_clickEditProfile_opensProfileListActivity`` is sufficient (no need for 'checks'. Ditto elsewhere in this suite & PR.

Reply:
I have removed 'checks' at certain places, wherever it was exceeding the max-line-length.